lesson02/assignment/index.html

```diff
@@ -4,17 +4,35 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
On line 1, include the following:
<!DOCTYPE html>

You need this as the first line in every HTML file to make it semantically correct.